src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -1732,11 +1740,18 @@ class LedgerRPC_test : public beast::unit_test::suite
         testLedgerEntryPayChan();
         testLedgerEntryRippleState();
         testLedgerEntryTicket();
-        testLedgerEntryUnknownOption();
         testLookupLedger();
         testNoQueue();
         testQueue();
         testLedgerAccountsOption();
+
+        // version specific tests
+        for (auto testVersion = RPC::apiMinimumSupportedVersion;
+             testVersion <= RPC::apiBetaVersion;
```

Comment:
Is there no maximum value, only apiBetaVersion? (i'm on a phone, can't easily check rn).

Reply:
`apiBetaVersion` is the maximum value.

Reply:
Yea, from my understanding `apiBetaVersion` >= `apiMaximumSupportedVersion ` is a true statement. However I've seen instances where we've done `std::max(apiBetaVersion, apiMaximumSupportedVersion)` (https://github.com/XRPLF/rippled/blob/11e914fbe97184e0566085948b04942a8301ac36/src/test/rpc/Version_test.cpp#L86)